File: Words-iOS/Words/Service/WordService.swift
import Foundation
import Accounts
import Social
import ReactiveCocoa


class WordService {

    init() {}

    func getWords(month: Int, year: Int) -> SignalProducer<WordResponse, NSError> {
        let words: [Word] = [
            Word(id: 1, word: "Cat", day: 1, month: 1, year: 2015, imageUrl: "http://truestorieswithgill.com/wp-content/uploads/2013/09/20130915-190532.jpg"),
            Word(id: 2, word: "Dog", day: 2, month: 1, year: 2015, imageUrl:"https://media3.giphy.com/media/10we3R8dLZQ7hS/200_s.gif"),
            Word(id: 3, word: "Bacon", day: 3, month: 1, year: 2015, imageUrl:"http://rs1img.memecdn.com/Bacon_o_137966.jpg"),
            Word(id: 4, word: "Train", day: 4, month: 1, year: 2015, imageUrl:"http://rs2img.memecdn.com/nyan-train_o_289564.jpg"),
            Word(id: 5, word: "Onion", day: 5, month: 1, year: 2015, imageUrl:"http://rs1img.memecdn.com/sinister-onion_o_2425497.jpg")]

        return SignalProducer(value: WordResponse(values: words))
            |> delay(1.0, onScheduler: QueueScheduler(queue: dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_LOW, 0), name: ""))
    }

}

File: Words-Android/app/src/main/java/com/example/al333z/words/service/WordService.java
package com.example.al333z.words.service;

import java.util.concurrent.TimeUnit;

import rx.Observable;

/**
 * Created by al333z on 14/06/15.
 */
public class WordService {

    public WordService() {
    }

    public Observable<WordResponse> getWords(int month, int year) {
        final Word[] words = {
                new Word(1, "Cat", 1, 1, 2015, "http://truestorieswithgill.com/wp-content/uploads/2013/09/20130915-190532.jpg"),
                new Word(2, "Dog", 2, 1, 2015, "https://media3.giphy.com/media/10we3R8dLZQ7hS/200_s.gif"),
                new Word(3, "Bacon", 3, 1, 2015, "http://rs1img.memecdn.com/Bacon_o_137966.jpg"),
                new Word(4, "Train", 4, 1, 2015, "http://rs2img.memecdn.com/nyan-train_o_289564.jpg"),
                new Word(5, "Onion", 5, 1, 2015, "http://rs1img.memecdn.com/sinister-onion_o_2425497.jpg")};

        return Observable.just(new WordResponse(words)).delay(1, TimeUnit.SECONDS);
    }

}
